src/gorilla.c

```diff
@@ -318,9 +318,7 @@ static ChunkResult appendFloat(CompressedChunk *chunk, double value) {
     u_int64_t trailing = TrailingZeros64(xorWithPrevious);
   
     // Prevent over flow of DOUBLE_LEADING
-    if (leading > 31) { 
-        leading = 31; 
-    }
+    if (leading > 31) { leading = 31; }
```

Comment:
@ashtul I prefer to always have multi-line ifs, its easier to read when going through big chunks of code.

Reply:
It is used sometimes and here there is an empty line before and after.